src/lib.rs

```diff
@@ -58,6 +58,10 @@
 /// let limit = Some(42);
 /// let order_dir = OrderDirection::Ascending;
 ///
+/// REVIEW: I would love to get a few more examples or a bit more docs on how the runtime binding
+/// REVIEW: substitution works. I.e. what is the meaning of the names (`{limit}`). How are those
+/// REVIEW: handled. This is the part I'm having most problems to wrap my head around.
```

Comment:
While I don't mind more examples, that's supposed to be explained by

https://github.com/kyrias/sqlx-conditional-queries/blob/801f69eceeeaa686ba148a339e1aad520d4864af/src/lib.rs#L21-L23